config.in

```diff
@@ -164,16 +164,16 @@ mode "resize" {
     # right will grow the containers width
     # up will shrink the containers height
     # down will grow the containers height
-    bindsym $left resize shrink width 10 px or 10 ppt
-    bindsym $down resize grow height 10 px or 10 ppt
-    bindsym $up resize shrink height 10 px or 10 ppt
-    bindsym $right resize grow width 10 px or 10 ppt
```

Comment:
These are in the [default i3 config](https://github.com/i3/i3/blob/2f29fce801985464e1d58120b3c2835e39c3e68a/etc/config#L160). Is there a reason that sway will not be supporting the syntax?

Reply:
What is this supposed to do? It's not documented on the i3 user manual.

Reply:
For tiling containers, it uses <amount> ppt, and for floating, it uses <amount> px.

Also, the syntax is in the i3 user guide under [section 6.12](https://i3wm.org/docs/userguide.html#resizingconfig):
`resize grow|shrink <direction> [<px> px [or <ppt> ppt]]`

Reply:
It includes it in the command syntax but doesn't explain how it works. I updated #1737 to include this.

Also, `resize grow|shrink <direction> <ppt> ppt` is undocumented by sway but also works.